src/dbnode/storage/namespace/schema.go

```diff
@@ -194,9 +206,9 @@ func loadFileDescriptorSet(fdSet *nsproto.FileDescriptorSet, msgName string) (*s
 
 	md := curfd.FindMessage(msgName)
 	if md != nil {
-		return &schemaDescr{deployId: fdSet.DeployId, md: md}, nil
+		return &schemaDescr{deployId: fdSet.DeployId, prevDeployId: fdSet.PrevId, md: md}, nil
```

Comment:
might be worth a constructor function at this point to make sure we're always creating `schemaDescr` correctly

Reply:
ok